### PR TITLE
Birth and movement quick form blog posts

### PR DESCRIPTION
**Pull request by mstenta**
I wrote up two blog posts to demonstrate the new Birth and Movement quick forms that will be included with farmOS v2.1.0.

Opening this as a draft to start, for initial review. I'll mark it as ready once v2.1.0 is released.

### CODE REVIEW

**Comment by mstenta** [comment]
The embedded MP4s are not working... still debugging... I think this is the first time we've tried using them in the new farmOS.org Gatsby.js build process. Might need to add support for them somehow.

**Comment by mstenta** [comment]
> The embedded MP4s are not working...

Might need to wait until this is figured out: https://github.com/farmOS/farmOS.org/issues/75

File: posts/2023/birth-quick-form/index.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+
+# Birth Quick Form
+
+The farmOS [v2.1.0 release](https://github.com/farmOS/farmOS/releases/tag/2.1.0)
+includes a new and improved *Birth Quick Form* for quickly and easily recording
+animal births. This feature existed in farmOS v1, and has now been ported to v2.
+
+See the video below for a quick demonstration, and read on for more details.
+
+<video autoplay control loop style="width: 100%; height: auto;">
```
Comment (wotnak) [inline posts/2023/birth-quick-form/index.md]: `style="width: 100%; height: auto;"`
Since these inline styles have to be repeated for every video element used in any blog post, it might be better to instead add them globally in the website styles https://github.com/farmOS/farmOS.org/blob/main/src/theme.js#L20-L34

Comment (mstenta) [reply posts/2023/birth-quick-form/index.md]: I had the same thought @wotnak. Perhaps I'll open a separate PR for that and depending on which of these gets merged first we can update this code.

Comment (mstenta) [reply posts/2023/birth-quick-form/index.md]: https://github.com/farmOS/farmOS.org/pull/83

File: posts/2023/birth-quick-form/index.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+  <source src="./birth.mp4" type="video/mp4">
+</video>
+
+Recording the birth of animals in farmOS requires creating a few different
+records:
+
+1. A mother Animal asset to represent the animal that is pregnant.
+2. A Birth log that represents when/where/how she gave birth.
+3. One or more Animal assets that represent the newly born children.
+4. One or more Observation logs to record the child birth weights.
```
Comment (symbioquine) [inline posts/2023/birth-quick-form/index.md]: This is arguably not required, but the sentence above this list implies it is... presumably the sentence above should say "may require" and this line should say "Zero or more Observation logs"

File: posts/2023/birth-quick-form/index.md [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+
+<video autoplay control loop style="width: 100%; height: auto;">
+  <source src="./birth.mp4" type="video/mp4">
+</video>
+
+Recording the birth of animals in farmOS requires creating a few different
+records:
+
+1. A mother Animal asset to represent the animal that is pregnant.
+2. A Birth log that represents when/where/how she gave birth.
```
Comment (symbioquine) [inline posts/2023/birth-quick-form/index.md]: I'm not totally sure, but it might be clearer (and more PC?) to `%s/she/the mother/`

File: posts/2023/movement-quick-form/index.md [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+Next, you must specify the assets that are moving. Start typing an asset's name
+into the Assets field and select the one you want from the dropdown that
+appears. Separate multiple assets with a comma.
+
+You can select the locations that the assets are moving to in the same way. If
+no locations are specified, this will clear the current location of the assets.
+
+If the assets and locations you selected have a geometry already, they will
+automatically show up on the map within the form. Current asset location
+geometry will be displayed in blue, and the location geometry will be displayed
```
Comment (symbioquine) [inline posts/2023/movement-quick-form/index.md]:
```suggestion
geometry will be displayed in blue, and the new geometry will be displayed
```

File: posts/2023/movement-quick-form/index.md [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+in orange.
+
+The location geometry is editable within the map, which allows you to customize
```
Comment (symbioquine) [inline posts/2023/movement-quick-form/index.md]:
```suggestion
The geometry for the movement is editable within the map, which allows you to customize
```

File: posts/2023/movement-quick-form/index.md [hunk 6]
```diff
@@ -0,0 +1,100 @@
...
+you want to edit, and use the modify, move, or delete buttons to change them.
+You can also use the polygon, line, point, and circle buttons to draw new
+geometries.
+
+Customizing the geometry of the movement does *not* change the geometry of the
+location assets you selected. The new geometry is saved to the movement log
+itself, which is what gets referenced when you look at the location of your
+assets. Asset location geometry is determined by its movement logs. For more
+information, see [Movements and location](https://farmos.org/guide/location/) in
+the farmOS user guide.
```
Comment (symbioquine) [inline posts/2023/movement-quick-form/index.md]: There's a lot of background packed in here which I think would be a little hard to parse for someone new to farmOS. I wonder if it would be better to simplify and lean more heavily on the official docs.

Something like...

```suggestion
Customizing the geometry of the movement does *not* change the geometry of the
location assets you selected. The custom geometry is only saved to the movement log
and thus only effects the moved asset(s). For more
information, see [Movements and location](https://farmos.org/guide/location/) in
the farmOS user guide.
```

File: posts/2023/movement-quick-form/index.md [hunk 7]
```diff
@@ -0,0 +1,100 @@
...
+
+At the bottom of form there is an area to enter notes about the movement, which
+will be saved to the "Notes" field of the log that is created.
+
+The "Completed" checkbox at the bottom defaults to "Completed", but can be
+unchecked if you are planning a future movement.
+
+Upon submitting the form, you will see a message with a link to the new log that
+is created, and if you look at your asset records you will see that their
+current location and geometry has been updated.
```
Comment (symbioquine) [inline posts/2023/movement-quick-form/index.md]:
```suggestion
is created, and if you look at your asset records you will see that their
current location and geometry have been updated.
```

?

**Review by symbioquine — DISMISSED** [review]
LGTM

**Comment by mstenta** [comment]
Thanks for the feedback @symbioquine! I made the changes you recommended in this fixup commit: https://github.com/farmOS/farmOS-community-blog/pull/16/commits/b5ebcf455db0a1cb6759893e7b40bc966ed13764

File: posts/2023/birth-quick-form/index.md [hunk 8]
```diff
@@ -0,0 +1,118 @@
...
+
+# Birth Quick Form
+
+The farmOS [v2.1.0 release](https://github.com/farmOS/farmOS/releases/tag/2.1.0)
+includes a new and improved *Birth Quick Form* for quickly and easily recording
+animal births. This feature existed in farmOS v1, and has now been ported to v2.
+
+See the video below for a quick demonstration, and read on for more details.
+
+<video autoplay control loop>
```
Comment (wotnak) [inline posts/2023/birth-quick-form/index.md]: Missing `s` in the control**s** attribute.
Autoplay videos are not the best idea when it comes to accessibility https://www.a11yproject.com/posts/never-use-auto-play/. I would remove the autoplay and loop attributes, so the reader can start the video when they come to the point in the post when it makes sense to watch it and clearly see when it ends.

Comment (mstenta) [reply posts/2023/birth-quick-form/index.md]: Ah good catch @wotnak! And great insight re: `autoplay` and `loop`. I pushed a fixup: https://github.com/farmOS/farmOS-community-blog/pull/16/commits/8b87f7ea3fa83e7d33ebc337f71a4f5b7d696328

**Review by wotnak — APPROVED** [review]

**Review by paul121 — APPROVED** [review]
